pkg/app/server/service/webservice/service.proto

```diff
@@ -438,8 +438,8 @@ message GetInsightDataRequest {
     int64 range_from = 2 [(validate.rules).int64.gt = 0];
     int64 range_to = 3 [(validate.rules).int64.gt = 0];
     string application_id = 4;
-    // tzdb Id like `Asia/Tokyo`. null means UTC.
-    string timezone = 5;
+    // seconds east of UTC
+    int32 offset = 5;
```

Comment:
should be int64 🤔

Reply:
A day is 86400 seconds, so I think int32 is sufficient (go [uses](https://pkg.go.dev/time#FixedZone) int)